src/app/globals.css

```
@@ -129,3 +129,51 @@
     @apply font-sans;
   }
 }
+
+/* Custom animations for tips page */
+@keyframes fadeIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)